spring-cloud-stream-core-docs/src/main/asciidoc/spring-cloud-stream-overview.adoc

```diff
@@ -1539,7 +1539,7 @@ For Spring Cloud Stream samples, please refer to the https://github.com/spring-c
 == Getting Started
 
 To get started with creating Spring Cloud Stream applications, visit the https://start.spring.io[Spring Initializr] and create a new Maven project named "GreetingSource".
-Select Spring Boot version 1.3.4 SNAPSHOT and search or tick the checkbox for Stream Kafka (we will be using Kafka for messaging).
+Select the Spring Boot version that works for you and search or tick the checkbox for Stream Kafka (we will be using Kafka for messaging).
```

Comment:
We should say only 1.4.x and 1.3.x since there are options for 2.0 snapshot and 1.5 snapshot and we may not work with those versions.